doc/nrf/test_and_optimize/optimizing/power_general.rst

```diff
@@ -181,11 +181,33 @@ Bluetooth Mesh
 
 The Bluetooth Mesh protocol offers the :ref:`ug_bt_mesh_configuring_lpn` feature for optimizing the power consumption of the Bluetooth Mesh devices.
 
+Gazell
+======
+
+Gazell is an asymmetrical protocol where the Device role is power-optimized.
+The exact current consumption will depend on a number of parameters, which are defined statically in the :ref:`nrfxlib:gzll_constants`.
```

Comment:
The definitions in "nrf_gzll_constants.h" file are provided as a reference. The users usually call GZLL API to modify the runtime parameters.

I am no expert in optimizing Gazell power consumption. Let me check with DevZone and get back to you.

Reply:
Thanks, then, once we have feedback form devzone I'll modify the text slightly. Are these values from "nrf_gzll_constants.h" the default ones, and then get modified through APIs in runtime?

Reply:
Yes, you get the right idea.